images/arm-img-builder.sh

```diff
@@ -182,6 +186,7 @@ while [ "$#" -gt 0 ]; do
     esac
     shift 1
 done
+load_vars
```

Comment:
aren't all the other cli flags override in this way except for the model? 
I've tried it locally and indeed: 
```
~/cos-chaser aarch_64firmware* ⇣⇡                                                                       
❯ sudo ./images/arm-img-builder.sh --local --directory /home/ettore/rpi-cos --size 900000
[sudo] password for root:                                                                               
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Image Size: 7544 MB.                     
Recovery Partition: 2192.                                                                               
State Partition: 4992 MB.
Images size (active/passive/recovery.img): 2400 MB.
Model: odroid_c2
Container image: quay.io/costoolkit/examples:odroid-c2-latest
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
>> Preparing active.img
2400+0 records in
2400+0 records out
2516582400 bytes (2.5 GB, 2.3 GiB) copied, 0.581036 s, 4.3 GB/s
mke2fs 1.46.4 (18-Aug-2021)
Discarding device blocks: done                             
Creating filesystem with 614400 4k blocks and 153824 inodes
Filesystem UUID: 4522bc12-98be-476d-a02a-74de527aa612
```

It didn't took neither the size neither the local directory